SocialCareCaseViewerApi/V1/UseCase/FormSubmissionsUseCase.cs

```diff
@@ -68,20 +68,20 @@ public FormSubmissionsUseCase(IDatabaseGateway databaseGateway, IMongoGateway mo
             return foundSubmission?.ToDomain().ToResponse();
         }
 
-        public void ExecuteDelete(string submissionId, DeleteCaseSubmissionRequest request)
+        public void ExecuteDelete(string submissionsId, DeleteCaseSubmissionRequest request)
```

Comment:
This will only ever handle one id, so would it be better to rename the parameter in the interface to be singular?

Reply:
Done